jenkins-pipelines/longevity-large-partition-200k-pks-4days-gce.jenkinsfile

```diff
@@ -7,6 +7,7 @@ longevityPipeline(
     backend: 'gce',
     region: 'us-east1',
     test_name: 'longevity_large_partition_test.LargePartitionLongevityTest.test_large_partition_longevity',
+    test_config: '''["test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml", "configurations/large-partition-200k-pks-4days-gce.yaml"]''',
```

Comment:
this entry is duplicate of next line, so you will be overriding this with the next line perhaps

Reply:
best to add this loader instance type to the test's configuration file, instead of creating a new file for it

Reply:
fixed.